node_modules/oae-authentication/config/strategies.js

```diff
@@ -58,6 +58,7 @@ module.exports = {
         'description': 'Allow Shibboleth authentication for tenant',
         'elements': {
             'enabled': new Fields.Bool('Shibboleth Authentication Enabled', 'Allow Shibboleth authentication for tenant', false),
+            'disableProfile': new Fields.Bool('Preventing modification of the name and mail', 'Preventing modification of the name and mail', true),
```

Comment:
@dooremont : Is it necessary to disable modification of the email as well? What if CAS has returned an invalid email address?